sessionctx/variable/sysvar.go

```diff
@@ -824,6 +825,14 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(s *SessionVars) (string, error) {
 		return BoolToOnOff(EnableNoopVariables.Load()), nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBAuthSigningCert, Value: "", Type: TypeStr, SetGlobal: func(s *SessionVars, val string) error {
+		sessionstates.SetCertPath(val)
+		return nil
+	}},
+	{Scope: ScopeGlobal, Name: TiDBAuthSigningKey, Value: "", Type: TypeStr, SetGlobal: func(s *SessionVars, val string) error {
+		sessionstates.SetKeyPath(val)
```

Comment:
At least for now, we could accept the identical cert path for all instances. TiDB instances with different paths like this would be hard to maintain and those paths are generally configured as command-line arguments.

BTW, this concern reminds me to confirm that: if we have multiple TiDB servers deployed on the same node, are they supposed to use the same certs and keys? @djshow832

Reply:
I talked about this with the PM. He suggests making it a global variable.

Reply:
> In MySQL paths are often relative to the data directory, which allows deployments on different paths. Maybe there should be a `CertDir` for this as well as other SSL/TLS related files?

I looked up the doc of SSL-related system variables in MySQL: https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html#using-encrypted-connections-server-side-runtime-configuration. I saw some examples from somewhere else and found that `ssl_ca`, `ssl_cert`, and `ssl_key` are not relative paths. Is that so?
A `CertDir` is a good idea, although it adds one more variable. The most important thing of updating the configuration of other TLS related files is backward compatibility. I think it's feasible.
I surveyed another DBMS as a reference: it has a `CertsDir` configuration. The file names under this directory should be organized as `client.<username>.crt`, `client-tenant.<tenant-id>.crt`, etc.

> Could/should we use something like AWS KMS for this?

I'm not familiar with KMS. I'll take a look. I'll appreciate much if you can summarize how to use it.